MetamodelLD.md

```diff
@@ -243,11 +243,12 @@ Deze paragraaf is een aanvulling op de paragraaf [[[#specificatie-metagegevens]]
 | indicatie formele historie | [`mim:indicatieFormeleHistorie`](http://bp4mc2.org/def/mim#indicatieFormeleHistorie) | owl:DatatypeProperty | [grondslag](#metagegeven-indicatie-formele-historie) |
 | informatiedomein | [`mim:informatiedomein`](http://bp4mc2.org/def/mim#informatiedomein) | owl:DatatypeProperty | [grondslag](#metagegeven-informatiedomein) |
 | informatiemodeltype | [`mim:informatiemodeltype`](http://bp4mc2.org/def/mim#informatiemodeltype) | owl:ObjectProperty | [grondslag](#metagegeven-informatiemodel-type) |
-| is gedefinieerd in | [`mim:isGedefinieerdIn`](http://bp4mc2.org/def/mim#isGedefinieerdIn) | owl:ObjectProperty | [grondslag](#metagegeven-is-gedefinieerd-in) |
+| is gedefinieerd in | [`mim:isGedefinieerdIn`](http://bp4mc2.org/def/mim#isGedefinieerdIn) | owl:DatatypeProperty | [grondslag](#metagegeven-is-gedefinieerd-in) |
```

Comment:
De betekenis van isGedefinieerdIn is de inverse van heeftModelement, behalve in situaties waarbij isGedefineerdIn expliciet is opgenomen. Maar het relateert een modelement met de package waarin het modelelement in gedefinieerd is. Dan zou het dus een owl:ObjectProperty moeten zijn en geen owl:DatatypeProperty.

Reply:
@pmaria Kan jij deze PR managen, afmaken en mergen?

Reply:
Yes